snap/snapcraft.yaml

```diff
@@ -43,20 +61,75 @@ parts:
   intel-graphics:
     plugin: nil
     stage-packages:
-      - vainfo
+      - clinfo
+      - intel-gsc
+      - intel-level-zero-gpu
       - intel-media-va-driver-non-free
-      - libvpl2
-      - libvpl-dev
+      - intel-metrics-discovery
+      - intel-ocloc
+      - intel-ocloc-dev
```

Comment:
Could you kindly remove the *-dev packages from the stage-packages list? These development packages are not needed at runtime and should not be staged in the final snap.